### ISSUE TITLE
I am receiving both real and test ads on IOS device on production how is this possible?

### ISSUE THREAD

**Issue by hacmak18** [kmcgill88/admob_flutter#254]
I am receiving ads on iOS devices and app is on AppStore but I am facing an issue of test ads. sometimes app shows real ads and sometimes it show test ads. link for the app is: https://apps.apple.com/app/id1545110534
is this related to this package or issue is something else?

**Comment by kmcgill88** [comment]
You didn't ship your app with test ad unit ids, did you? 🤷🏻‍♂️

**Comment by hacmak18** [comment]
No. I released application with real Ad ids. When i open app and click on button to show Ad, that works fine. But very next time when i click on the same button test ad appears.
Later on i found out issue raised whenever i try to show ads and without calling load method.

**Comment by am1tr0r** [comment]
@hacmak18  is it fixed now ? how did you fix this ?
